### ISSUE TITLE
libc10_cuda.so not found

### ISSUE THREAD

**Issue by nem25** [autogluon/autogluon#1671]
I just installed a fresh copy of autogluon on an Ubuntu 20.04 machine.
Python 3.8
When I try running the basic tutorial I get the following error:
```
/home/user/.local/lib/python3.8/site-packages/torchvision/io/image.py:13: UserWarning: Failed to load image Python extension: libc10_cuda.so: cannot open shared object file: No such file or directory
  warn(f"Failed to load image Python extension: {e}")
```

**Comment by Innixma** [comment]
Thanks for creating an issue! Please provide a full reproducible script along with more information on the machine, such as if it has a GPU.

**Comment by nem25** [comment]
Hi,

Apologies. I should have done that earlier.

This is the code that we tried:
```
import os
from autogluon.core.utils.loaders import load_pd
from autogluon.text import TextPredictor

train_data = load_pd.load('https://autogluon-text.s3-accelerate.amazonaws.com/glue/sst/train.parquet')

subsample_size = 1000  # subsample data for faster demo, try setting this to larger values
train_data = train_data.sample(n=subsample_size, random_state=0)

# train_data.head(10)

predictor = TextPredictor(label='label', eval_metric='acc', path='./sentiment1')
#Training the model. Time_limit is given in case it takes too long while testing, it will kill the training.
predictor.fit(train_data, time_limit=1000)
```

It doesn't even reach the GPU stage because I know that computer doesn't have any GPU options and generally when running the models I use the following flag: `os.environ["AUTOGLUON_TEXT_TRAIN_WITHOUT_GPU"]="1"`

Here is the output:
```
cd /home/user/Documents/autogluon-esg-main ; /usr/bin/env /bin/python3 /home/user/.vscode/extensions/ms-python.python-2022.4.1/pythonFiles/lib/python/debugpy/launcher 43483 -- /home/user/Documents/autogluon-esg-main/tutorial.py
/home/user/.local/lib/python3.8/site-packages/torchvision/io/image.py:13: UserWarning: Failed to load image Python extension: libc10_cuda.so: cannot open shared object file: No such file or directory
warn(f"Failed to load image Python extension: {e}")
```

I only started having this problem after installing the newer version of autogluon that had switched to pytorch.

**Comment by Innixma** [comment]
Hopefully fixed as of v0.6, and very likely fixed in v0.7 with autogluon.multimodal. Please re-open if problems persist in v0.7 or latest mainline.
